libs/interpreter-lib/src/interpreter.ts

```diff
@@ -158,7 +160,17 @@ function setupJayveeServices(
   );
 
   services.validation.ValidationRegistry.registerJayveeValidationChecks({
-    RuntimeParameterLiteral: validateRuntimeParameterLiteral,
+    RuntimeParameterLiteral: (
+      runtimeParameter: RuntimeParameterLiteral,
+      validationContext: ValidationContext,
+      evaluationContext: EvaluationContext,
+    ) =>
+      validateRuntimeParameterLiteral(
+        runtimeParameter,
+        validationContext,
+        evaluationContext,
+        services.WrapperFactory,
```

Comment:
Can be removed?

Reply:
Done in [1dac8ca](https://github.com/jvalue/jayvee/pull/536/commits/1dac8cafe69fb7e920efce243e7ca4c5220f0575)